apps/web/src/design-system/config/colors.ts

```diff
@@ -13,7 +13,7 @@ export const colors = {
   B15: '#1E1E26',
   B17: '#23232B',
   success: '#4D9980',
-  error: '#E54545',
```

Comment:
why are we changing this?

Reply:
I remember changing this after checking one component in Figma, but now that I checked it again seems somehow I made a mistake. Thanks for spotting it.